receiver/hostmetricsreceiver/internal/scraper/cpuscraper/cpu_scraper.go

```diff
@@ -24,24 +24,27 @@ import (
 	"go.opentelemetry.io/collector/model/pdata"
 	"go.opentelemetry.io/collector/receiver/scrapererror"
 
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/cpuscraper/internal"
```

Comment:
Maybe create a package specifically for the utilization calculator instead of reusing one shared with metadata? Something like this?

```suggestion
	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/cpuscraper/ucal"
```

And the whole scraper is under `internal` already anyway